### ISSUE TITLE
Add usage directions

### ISSUE THREAD

**Issue by bonelifer** [andremueller/session-notify#1]
Could you add installation and usage instructions to the  readme file?

**Comment by andremueller** [comment]
You are absolutely right. Even for myself its a good idea to add a bit of comments. Thanks for checking out my projects.
